code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -5419,6 +5488,199 @@
 		return
 	..()
 
+/obj/item/reagent_containers/food/snacks/earthenroot_mash
+	name = "mashed earthen-root"
+	desc = "Mounds of mashed earthen-root. Soft and pillowy."
+	icon_state = "earthenroot_mash"
+	trash = /obj/item/trash/plate
+	filling_color = "#3884a7"
+	reagents_to_add = list(/singleton/reagent/nutriment = 4)
+	reagent_data = list(/singleton/reagent/nutriment = list("mashed earthenroot" = 4))
+	bitesize = 1
+
+/obj/item/reagent_containers/food/snacks/earthenroot_chopped
+	name = "chopped earthen-root"
+	desc = "Chopped earthen-root. Cooking this would make it nicer."
+	icon_state = "earthenroot_chopped"
+	trash = /obj/item/trash/plate
+	filling_color = "#5498b8"
+	reagents_to_add = list(/singleton/reagent/nutriment = 3)
+	reagent_data = list(/singleton/reagent/nutriment = list("sweet, chunky earthenroot" = 3))
+	bitesize = 2
+
+/obj/item/reagent_containers/food/snacks/earthenroot_fries
+	name = "earthen-root fries"
+	desc = "A plate full of fresh earthen-root fries. A crispy sweet Adhomian treat."
+	icon_state = "earthenroot_fries"
+	trash = /obj/item/trash/plate
+	filling_color = "#3884a7"
+	reagents_to_add = list(/singleton/reagent/nutriment = 4, /singleton/reagent/nutriment/triglyceride/oil = 1.2)
+	reagent_data = list(/singleton/reagent/nutriment = list("crispy sweet earthen-root fries" = 4))
+	bitesize = 2
+
+/obj/item/reagent_containers/food/snacks/earthenroot_wedges
+	name = "fried earthen-root wedges"
+	desc = "Some fried wedges made from earthen-root. Chunkier than fries, but still crispy."
+	icon_state = "earthenroot_wedges"
+	trash = /obj/item/trash/plate
+	filling_color = "#3884a7"
+	reagents_to_add = list(/singleton/reagent/nutriment = 4, /singleton/reagent/nutriment/triglyceride/oil = 1.2)
+	reagent_data = list(/singleton/reagent/nutriment = list("crispy sweet earthen-root wedges" = 4))
+	bitesize = 2
+
+/obj/item/reagent_containers/food/snacks/salad/earthenroot
+	name = "earthen-root salad"
+	desc = "A Tajaran salad containing earthen-root, sarmikhir, herbs and cream. A great option for the more healthy-minded."
+	icon_state = "earthenroot_salad"
+	trash = /obj/item/trash/snack_bowl
+	reagents_to_add = list(/singleton/reagent/nutriment = 10)
+	reagent_data = list(/singleton/reagent/nutriment = list("sweet chunky earthen-root" = 2, "sour cream" = 2, "oily berries" = 2, "sweet herbs" = 2))
+	bitesize = 3
+
+/obj/item/reagent_containers/food/snacks/stuffed_earthenroot
+	name = "stuffed earthen-root"
+	desc = "An earthen-root stuffed with adhomian meat. Crunchy on the outside, savoury on the inside."
```

Comment:
```suggestion
	desc = "An earthen-root stuffed with Adhomian meat. Crunchy on the outside, savoury on the inside."
```
Adhomai is a proper noun, and its denotation should follow suit.

Reply:
fixed